src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -298,6 +298,16 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
           <Localized id="MenuButtons--metaInfo--platform">Platform</Localized>
         </h2>
         <div className="metaInfoSection">
+          {meta.device ? (
+            <div className="metaInfoRow">
```

Comment:
Suggestion: This could use a snapshot test since it's uncovered.

Reply:
Actually doing a snapshot test plus a `getByText("Android")` combo would be even better.